docs/prometheusoperator.md

```diff
@@ -123,29 +124,28 @@ endpoints:
 
 If the Coherence helm chart parameter `service.metricsHttpPort` is set when installing the Coherence helm chart,
 replace `9095` above with the new value.
-  
-If the Coherence helm chart parameter `store.metrics.ssl.enabled` is `true`, additionally add `endpoints.scheme` value of `https`
-to `coherence-service-monitor.yaml` fragment.
-    
+
+If the Coherence helm chart parameter `store.metrics.ssl.enabled` is set to `true`, then add  `endpoints.scheme` with the value of `https`
+to the `coherence-service-monitor.yaml` fragment.
+
 ## Troubleshooting
 
-## Helm install of `coherence-operator` fails creating a custom resource definition (CRD).
```

Comment:
retain the heading as Coherence-operator fails creating a custom resource definition (CRD)
Start the section like this
When the 'helm install' of coherence-operator fails create to create CRDs, follow the recommendation.......

Reply:
Fixed the titles of troublehshooting errors.